public/static/partials/subscription_list.html

```diff
@@ -14,6 +14,10 @@ <h4>Subscriptions
             <button class="pull-right plus-button btn btn-primary" type="button" ng-click="addSubscription()">
                 <i class="fa fa-plus"></i>
             </button>
+            <div class="muted small">
```

Comment:
Can we add a comment with the ticket number that we're going to fix this in? (also to the spruce PR)

Reply:
for sure